executor/write.go

```diff
@@ -1594,11 +1592,11 @@ func (e *InsertValues) fillGenColData(cols []*table.Column, valLen int, hasValue
 	return row, nil
 }
 
-func (e *InsertValues) filterErr(err error, ignoreErr bool) error {
+func (e *InsertValues) filterErr(err error) error {
 	if err == nil {
 		return nil
 	}
-	if !ignoreErr {
+	if !e.ctx.GetSessionVars().StmtCtx.IgnoreErr {
```

Comment:
We can merge this check with line 1596

if err !== nil || ....{
    return errors.Trace(err)
}

Reply:
It will be refactored in the next pr.